resources/ahk/TradeMacro.ahk

```diff
@@ -87,13 +87,6 @@ OpenSearchOnPoeTrade:
 	SuspendPOEItemScript = 1 ; This allows us to handle the clipboard change event
 	Send ^{sc02E}
 	Sleep 250
-	
-	TradeFunc_DoParseClipboard()
-	If (!Item.Name and TradeOpts.OpenUrlsOnEmptyItem) {
```

Comment:
That part is not even related to the wiki, its within  OpenSearchOnPoeTrade: ...
and it was redundant since the exact same lines are are the beginning of TradeFunc_Main()
```

OpenSearchOnPoeTrade:
	Global TradeOpts, Item
	Item := {}
	SuspendPOEItemScript = 1 ; This allows us to handle the clipboard change event
	Send ^{sc02E}
	Sleep 250
	
	TradeFunc_DoParseClipboard()
	If (!Item.Name and TradeOpts.OpenUrlsOnEmptyItem) {
		TradeFunc_OpenUrlInBrowser("http://poe.trade/")
		return
	}
	
	TradeFunc_Main(true)
	SuspendPOEItemScript = 0 ; Allow Item info to handle clipboard change event
return

; Prepare Reqeust Parametes and send Post Request
; openSearchInBrowser : set to true to open the search on poe.trade instead of showing the tooltip
; isAdvancedPriceCheck : set to true If the GUI to select mods should be openend
; isAdvancedPriceCheckRedirect : set to true If the search is triggered from the GUI
; isItemAgeRequest : set to true to check own listed items age
TradeFunc_Main(openSearchInBrowser = false, isAdvancedPriceCheck = false, isAdvancedPriceCheckRedirect = false, isItemAgeRequest = false)
{	
	LeagueName := TradeGlobals.Get("LeagueName")
	Global Item, ItemData, TradeOpts, mapList, uniqueMapList, Opts
		
	TradeFunc_DoParseClipboard()
	iLvl     := Item.Level
	
	; cancel search If Item is empty
	If (!Item.name) {
		If (TradeOpts.OpenUrlsOnEmptyItem) {
			TradeFunc_OpenUrlInBrowser("https://poe.trade")
		}
		return
	}
	
```